logrotate.d/ovirt-dr.in

```diff
@@ -0,0 +1,8 @@
+"@PKG_LOG_DIR/LOG_NAME@/ovirt-dr.log" {
```

Comment:
Where are you replacing `@PKG_LOG_DIR/LOG_NAME@` with `/var/log/ovirt-dr`? Also why is there only single log file? Where are you going to log multiple concurrent invocations of a role from single machine?

Reply:
That is what I've defined in the spec, I will try to do that with '/var/log/ovirt-dr'
I've tried to follow how ovirt-engine works, should I do it like this:
  "/var/log/ovirt-dr"/*.log

Reply:
Is it ok if the file is located here:  logrotate.d/ovirt-dr.in